polar-embed.md

```diff
@@ -180,6 +182,13 @@ html: boolean
 ```
 This defines wether the HTML of generated page will be returned in result or not.
 
+### CSS selector prefix:
+```TypeScript
+cssSelectorPrefix: string
+```
+A CSS selector string that will be prefixed to all style selectors in the generated CSS.
```

Comment:
This needs a bit of adjustment. Only the CSS Rules generated by the polar Popup will be prefixed, but no the CSS rules passed in through the CSS property.
In current state, the `.fr_header {background-color: #fff;}` will have the same `.fr_header` class in the output without any prefixes.